src/index.ts

```diff
@@ -41,45 +75,95 @@ app.put("/publish/:id", async (req, res) => {
 app.delete(`/post/:id`, async (req, res) => {
   //delete the post
   const { id } = req.params;
-  const post = await prisma;
+  const post = await prisma.post.delete({
+    where: {
+      id: Number(id),
+    },
+  });
   res.json(post);
 });
 
 app.get("/users", async (req, res) => {
   //return all the users
-  const users = await prisma;
+  const users = await prisma.user.findMany();
   res.json(users);
 });
 
 app.get("/user/:id/drafts", async (req, res) => {
   const { id } = req.params;
   //return all posts where the published field equals false
-  const drafts = await prisma;
+  const drafts = await prisma.user.findMany({
```

Comment:
`findMany` is not the correct method since there is only one user for a `id`. Instead find one unique user and use the posts method on the user item